app/views/list/_table.html.erb

```diff
@@ -0,0 +1,21 @@
+<table class="govuk-table">
+  <thead class="govuk-table__head">
+    <tr class="govuk-table__row">
+      <th class="govuk-table__header">Name</th>
+      <th class="govuk-table__header">Created At</th>
+      <th class="govuk-table__header">Updated At</th>
+      <th class="govuk-table__header">Service Email</th>
+    </tr>
+  </thead>
+  <tbody class="govuk-table__body">
+        <tr class="govuk-table__row">
+  <% @all_organisations.each do | organisation| %>
+        <td class="govuk-table__cell quarter-column" scope="row"><%=  organisation.name %></td>
+        <td class="govuk-table__cell quarter-column" scope="row"><%=  organisation.created_at.strftime('%e %b %Y %H:%M %P') %></td>
```

Comment:
What does the  `strftime` mean and what does the `('%e %b %Y %H:%M %P')` do?

Reply:
It's a method that formats the date and time. The arguments are what determine how it looks.